activemq/README.md

```diff
@@ -81,7 +81,7 @@ For containerized environments, see the [Autodiscovery Integration Templates][10
 | Parameter            | Value                                |
 | -------------------- | ------------------------------------ |
 | `<INTEGRATION_NAME>` | `activemq`                           |
-| `<INIT_CONFIG>`      | blank or `{}`                        |
+| `<INIT_CONFIG>`      | `is_jmx:true` or `{}`                |
```

Comment:
```suggestion
| `<INIT_CONFIG>`      | `{"is_jmx": "true"}`               |
```

I think we should drop the empty `{}` without `is_jmx` the check won't work. I think the correct format would be `{"is_jmx": "true"}`. But looking at other parts of the docs, it seems that way.

Reply:
@AxelleFesard Do you have any opinions as to this? I'm not familiar with this integration, so I need to defer to you and @steveny91 on this.

Reply:
Hello!! I think that @steveny91 is right. We should not have the option {} for active_mq check init_config indeed. 
And I would have use the same syntax as well for the annotations as we can see here: https://docs.datadoghq.com/containers/guide/autodiscovery-with-jmx/?tab=helm
https://a.cl.ly/X6u7yGNG
![Image 2023-08-16 at 4 06 51 PM](https://github.com/DataDog/integrations-core/assets/111456392/38ef6f19-2de2-4464-a115-19420ac995c6)